haste/windowed_conformal_model/time_series_features.py

```diff
@@ -28,15 +26,12 @@ def time_series_features(feature_time_series, timestamps, end_time):
 
     feat = np.zeros(4)
 
-    X = pd.DataFrame(timestamps)
-    Y = pd.Series(feature_time_series)
+    feat[0] = np.mean(feature_time_series)  # mean of raw data
+    feat[1] = np.std(feature_time_series)  # standard deviation of raw data
 
-    gam = LinearGAM(n_splines=min(25, len(Y))).gridsearch(X, Y)
-
-    feat[0] = np.mean(Y)  # mean of raw data
-    feat[1] = np.std(Y)  # standard deviation of raw data
-
-    y_pred = gam.predict(pd.DataFrame(np.arange(end_time)))
+    z = np.polyfit(timestamps, feature_time_series, min(8, round(len(timestamps)/3)))
```

Comment:
is the 8 here the window size? can we reference the constant?

Reply:
No, it's a separate thing. So it stays as 8 in all cases for the time being.

Reply:
min(8, round(len(timestamps)/3) is what degree of the polynomial we want to fit so its kind of a "hack" until we have a better solution, but it works for now